apps/server/lib/lexical/server/code_intelligence/completion/translations/module_attribute.ex

```diff
@@ -59,10 +59,21 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.ModuleAttribut
   end
 
   def translate(%Candidate.ModuleAttribute{} = attribute, builder, env) do
-    builder.plain_text(env, attribute.name,
+    builder.text_edit(env, attribute.name, range(env),
       detail: "module attribute",
       kind: :constant,
       label: attribute.name
     )
   end
+
+  defp range(%Env{} = env) do
+    case Env.prefix_tokens(env, 1) do
+      [{:at_op, _, {_line, char}}] ->
+        {char, env.position.character}
+
+      # ensure we include the @ operator in the text edit range
```

Comment:
this comment is confusing to me. It looks like the case clause at line 71 is the one that has an @ operator.

Reply:
Refactored this function to be more robust and clear.